src/CUDAKernels.jl

```diff
@@ -243,4 +243,10 @@ function KA.priority!(::CUDABackend, prio::Symbol)
     return nothing
 end
 
+KA.supports_enzyme(::CUDABackend) = true
```

Comment:
It seems weird/wrong to add an Enzyme-specific API to the KA.jl interface, while the Enzyme support is all in an extension packages.